src/packages/features/index.ts

```diff
@@ -2,5 +2,6 @@ export default {
   CONSOLE_LISTENER: 'perry::feature::consolelistener',
   DOCUMENT_CLICK_LISTENER: 'perry::feature::documentclicklistener',
   WINDOW_ERROR_LISTENER: 'perry::feature::windowerrorlistener',
-  NOTIFY_LISTENER:'perry::feature::notifylistener'
+  NOTIFY_LISTENER:'perry::feature::notifylistener',
+  WINDOW_SCREEN_RECORD: 'perry::feature::windowsreenrecord',
```

Comment:
minor, but there is a small typo: `windowsreenrecord` should be `windowscreenrecord` I believe :P